### PR TITLE
Changed login text to be less misleading

### LINKED ISSUE #5053: "Do you want to save this login?" is misleading.
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

The popup when you login says "**Do you want to save this login?**".

This is misleading, as it can lead you to believe it will save the login credentials. Maybe you think it is your password manager or your browser asking the question, and not "Home Assistant", which is what I did. This could then cause you to loose your password, and the next time you are logged out you will discover that you are locked out of your account because the login credentials wasn't actually saved anywhere. That's the situation I was faced with.

I could provide  a screenshot, but it probably isn't necessary to understand what I mean here.
The popup comes in the lower right corner of the browser.

## Expected behavior

The popup should say something like "**Do you want this login session to persist through page refreshes?**". Which gives precise information about what it actually does, and why you would want to say yes to it. The button should correspondingly say "Yes" and not "Save".

## Steps to reproduce

Go to profile page, log out, and then log back in again. See the popup in the bottom right corner.

## Environment

- Home Assistant release with the issue: 0.106.4
- Last working Home Assistant release (if known): 
- UI Type (States or Lovelace): 
- Browser and browser version: Chrome Version 80.0.3987.122 (Official Build) (64-bit)
- Operating system: macOS High Sierra Version 10.13.6

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### PR DESCRIPTION
**Pull request by redbar0n**
See https://github.com/home-assistant/frontend/issues/5053

## Proposed change

Changed the login text to be less misleading and more precise. As it were, it could lead you to believe the login credentials were saved, when it was actually merely the session that was persistend. The misleading text caused confusion, bug requests, and potentially users locked out of their accounts. A very small fix.

See: https://github.com/home-assistant/frontend/issues/5053

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5053
- This PR is related to issue: #157, #3478
- Link to documentation pull request: 

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @redbar0n, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;frontend#5054).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: src/translations/en.json [hunk 1]
```diff
@@ -377,9 +377,9 @@
   },
   "ui": {
     "auth_store": {
-      "ask": "Do you want to save this login?",
-      "decline": "No thanks",
-      "confirm": "Save login"
+      "ask": "Do you want this login session to persist?",
```
Comment (bramkragten) [inline src/translations/en.json]: Maybe: `Do you want to stay logged in?`

Comment (zsarnett) [reply src/translations/en.json]: Agree with Brams Suggestion

Comment (redbar0n) [reply src/translations/en.json]: Yes! Even better :D

**Pull request merged**